Makefile

```
@@ -0,0 +1,112 @@
+# ─── StablePay Payment Pipeline — Makefile ──────────────────────────────────
+# Convenience targets for local development. The canonical tool is `just`
+# (see justfile); this Makefile provides make-based equivalents.
+
+COMPOSE := docker compose -f infra/docker-compose.yml
+COMPOSE_CI := docker compose -f infra/docker-compose.yml -f infra/docker-compose.ci.yml
+
+.PHONY: help up down nuke logs status \
+        build-api build-auth build-web build-all \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)